describealign.py

```diff
@@ -1195,8 +1195,8 @@ def command_line_interface():
   parser = argparse.ArgumentParser(
                           description="Replaces a video's sound with an audio description.",
                           usage="describealign video_file.mp4 audio_file.mp3")
-  parser.add_argument("video", help='A video file or directory containing video files.', default=None)
-  parser.add_argument("audio", help='An audio file or directory containing audio files.', default=None)
+  parser.add_argument("video", help='A video file or directory containing video files.', nargs='?')
```

Comment:
```python
>>> import argparse
>>> parser = argparse.ArgumentParser()
>>> parser.add_argument('foo', nargs='?')
_StoreAction(option_strings=[], dest='foo', nargs='?', const=None, default=None, type=None, choices=None, required=False, help=None, metavar=None)
>>> parser.parse_args([])
Namespace(foo=None)
>>> parser.parse_args(['bar'])
Namespace(foo='bar')
```

The default for any argument is `None` unless otherwise specified with `default=`, and setting `default=` on a positional argument is an error.

Reply:
Setting the default value as None doesn't give an error for me:
```
import argparse

parser = argparse.ArgumentParser()
parser.add_argument('blah', nargs='?', help='Blah.', default=None)
parser.add_argument('blah2', nargs='?', help='Blah2.', default=None)

print(parser.parse_args(['in', 'in2']))
print(parser.parse_args(['in']))
print(parser.parse_args([]))
```
I'd rather be explicit about the default value, but not a big deal if you think it's cleaner without it

Reply:
Ah yep I misremembered my testing, setting `optional` fails, not `default`. Yep, 100% agree with being more explicit, thanks for catching this!